libs/editor/src/lib/components/wizard/wizard.component.ts

```diff
@@ -51,7 +55,17 @@ export class WizardComponent implements OnInit, AfterViewInit {
     this.stepChanges.emit(this.currentStep)
   }
 
-  handleNextBtnClick() {
+  async handleNextBtnClick() {
```

Comment:
why using a promise ?

Reply:
The use of `await` inside the function require it to be async.

Reply:
Yes but why to you use `await` inside the function ?
```js
this.translate.get('datafeeder.wizard.emptyRequiredValuesMessage').pipe(take(1))
   .subscribe( msg => alert(msg} )
```